contrib/runners/python_runner/python_runner/python_action_wrapper.py

```diff
@@ -28,6 +28,12 @@
 
 import distutils.sysconfig
 
+# NOTE: We intentionally use orjson directly here instead of json_encode - orjson.dumps relies
+# on config option which we don't parse for the action wrapper since it speeds things down - action
+# wrapper should rely on as little imports as possible to make Python runner executions fast -
+# that's very important.
+import orjson
```

Comment:
Could we use jsonify json encode/decode methods here instead of referencing orjson directly here?

Reply:
EDIT: Ignore my comment above, I thought I was looking at another PR.

Actually, no, we can't do that because of the comment above the import line.

> 
> NOTE: We intentionally use orjson directly here instead of json_encode - orjson.dumps relies
> on config option which we don't parse for the action wrapper since it speeds things down - action
> wrapper should rely on as little imports as possible to make Python runner executions fast -
> that's very important.

In short, in python action wrapper every 10s of milliseconds make a difference and importing that module adds a bunch of overhead since it has import time side affects due to loading oslo config, etc.

Reply:
Ack.